ci/cscs.yml

```diff
@@ -0,0 +1,45 @@
+include:
+  - remote: 'https://gitlab.com/cscs-ci/recipes/-/raw/master/templates/v2/.ci-ext.yml'
+
+stages:
+  - prepare
+  - configure
+  - run
+
+variables:
+  CI_GENERATOR_IMAGE_NAME: $CSCS_REGISTRY_PATH/public/ci-generator:$CI_GENERATOR_TAG
```

Comment:
I would suggest to have this `CI_GENERATOR_IMAGE_NAME: $CSCS_REGISTRY_PATH/public/ci-generator`